src/helpers/amoc_xmpp_handlers.erl

```diff
@@ -23,9 +23,12 @@
 %% Types
 -type handler_spec() :: {escalus_connection:stanza_pred(), action()}.
 
+%% TODO: export this type from escalus
```

Comment:
was `escalus_connection:metadata()` type unexported?

Reply:
Yeap, it is unexported. I didn't want to block this one by yet another PR in escalus to export it and so on, but that's a pending task there 🥲